test/unit/auth/user-record.spec.ts

```diff
@@ -204,11 +239,270 @@ function getUserInfoWithPhoneNumberJSON(): object {
   };
 }
 
+describe('PhoneMultiFactorInfo', () => {
+  const serverResponse: any = {
+    mfaEnrollmentId: 'enrollmentId1',
+    displayName: 'displayName1',
+    enrolledAt: now.toISOString(),
+    phoneInfo: '+16505551234',
+  };
+  const phoneMultiFactorInfo = new PhoneMultiFactorInfo(serverResponse);
+  const phoneMultiFactorInfoMissingFields = new PhoneMultiFactorInfo({
+    mfaEnrollmentId: serverResponse.mfaEnrollmentId,
+    phoneInfo: serverResponse.phoneInfo,
+  });
+
+  describe('constructor', () => {
+    it('should throw when an empty object is provided', () => {
+      expect(() =>  {
+        return new PhoneMultiFactorInfo({} as any);
+      }).to.throw(Error);
```

Comment:
Can we test for a specific error message?

Reply:
Done.